content/en/docs/usage/gateway-api.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Securing Gateway Resources"
+linkTitle: "Securing Gateway Resources"
+weight: 90
+type: "docs"
+---
+
+Since 1.5, cert-manager supports requesting TLS certificates using annotations
+on Gateway resources. The Gateway resource is part of the [Gateway API][gwapi],
+a set of CRDs that can be installed to your Kubernetes cluster and that aim at
+offering an richer alternative to the Ingress abstraction.
+
+[gwapi]: https://gateway-api.sigs.k8s.io
+
+The Gateway resource holds the TLS configuration, as illustrated in the
+following diagram (source: https://gateway-api.sigs.k8s.io):
+
+![](https://gateway-api.sigs.k8s.io/images/gateway-roles.png)
+
+Note that cert-manager only supports setting up the TLS configuration on the
+Gateway resource when the Gateway is configured to terminate the TLS connection.
+There is still uncertainty around how the HTTPRoute TLS configuration should be
+used (see
+[gateway-api#577](https://github.com/kubernetes-sigs/gateway-api/issues/577)),
+and no existing implementation supports the TLS passthrough mode in which the
+HTTPRoute contains the TLS configuration and terminates the connection (see
+[istio#31747](https://github.com/istio/istio/issues/31747)).
+
+The Gateway support for Certificate shims must be enabled manually using a flag.
+If you are using Helm:
+
+```sh
+helm upgrade --install cert-manager jetstack/cert-manager
+  --set "extraArgs={--controllers=*\,gateway-shim}"
+```
+
+If you are using the raw cert-manager manifests, you can add the following like
+to the cert-manager controller Deployment `args`:
+
+```sh
+--controllers=*,gateway-shim
+```
+
+Like with an Ingress, the annotations `cert-manager.io/issuer` or
+`cert-manager.io/cluster-issuer` tells cert-manager that this Gateway should be
+looked at to create Certificate shims. For example, the following Gateway should
```

Comment:
Suggestion: here and elsewhere, I think arguably you could call them just `Certificate`s instead of 'Certificate shims' to not confuse folks since they will then work in the same way as `Certificate`s.

Reply:
Good idea, "shim" is confusing.